ci/stage/elasticsearch_s3backup.Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env groovy
+/**
+  This is the Jenkinsfile for backing up elasticsearch using s3.
+  Elasticsearch S3 repository plugin is required.
+ */
+ansiColor('xterm') {
+  timestamps {
+        node('dhslave') {
+          configFileProvider(
+          [configFile( fileId: 'kubeconfig', variable: 'KUBECONFIG')]){
+	      stage('Pull Git Repositories') {
+                checkout_playbook_repo()
+              }
+              stage('Trigger Playbook') {
+                start_backup()
+              }
+          }
+        }
+  }
+}
+def checkout_playbook_repo() {
+    checkout poll: false, scm: [
+      $class: 'GitSCM',
+      branches: [[name: "*/master"]],
+      doGenerateSubmoduleConfigurations: false,
+      extensions: [
+        [$class: 'WipeWorkspace'],
+        [$class: 'RelativeTargetDirectory', relativeTargetDir: 'dh-ci-util']
+      ],
+      submoduleCfg: [],
+      userRemoteConfigs: [[url: 'https://gitlab.cee.redhat.com/data-hub/dh-ci-util.git']]
+    ]
+}
+def start_backup() {
+  try {
+    // Check if repository exists
+    checkRepositoryResponse = sh (
+      script: 'curl --insecure --cert $WORKSPACE/dh-ci-util/certificates/stage-es-certs/system.admin.crt --key $WORKSPACE/dh-ci-util/certificates/stage-es-certs/system.admin.key -i -k ${ELASTICSEARCH_ENDPOINT}/_snapshot/${BACKUP_NAME}',
```

Comment:
where are the variables `ELASTICSEARCH_ENDPOINT`, `BACKUP_NAME` coming from?

Reply:
These will be passed in via the Jenkins file.  We can create a story for the JJB part of it.